internals/generators/container/index.js

```diff
@@ -44,6 +44,11 @@ module.exports = {
     name: 'wantMessages',
     default: true,
     message: 'Do you want i18n messages (i.e. will this component use text)?',
+  }, {
+    type: 'confirm',
+    name: 'wantLoadable',
+    default: true,
+    message: 'Do you want an async loader?',
```

Comment:
Thanks for reminding me that loader is necessary if we want reducers/sagas, very helpful! I completely redone generators, now `Loadable.js` is always added if a dev wants reducers or sagas.